server/controllers/eventController.js

```diff
@@ -0,0 +1,74 @@
+const Event = require("../models/eventModel");
+
+// Example of creating a document in the database
+const createEvent = async (req, res) => {
+  const event = new Event(req.body);
+  try{
+    const data = await event.save(event);
+    res.send(data);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const getEventById = async (req, res) => {
+  const eventId = req.params.id;
+  try {
+    const event = await Event.findById(eventId);
+    res.send(event);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const getAllEvents = async (res) =>
+{
+  try {
+    const events = await Event.find();
+    res.send(events);
+  } catch (err) {
+    console.error(err);
+  }
+}
```

Comment:
Given any express function, there will always be 3 variables: `req` for the request object, `res` for the response object and `next` to call the next part of the route (if any). In this case, you only have one variable, which you label as `res` but in reality, it is `req` due to the ordering of the parameters.

This causes the backend to fail to send a response, despite correctly retrieving the events from the MongoDB. Please fix, thanks!